### PR TITLE
Polkadart Milestone 3

### PR DESCRIPTION
**Pull request by leonardocustodio**
# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [x] This pull request is being made by the same account as the accepted application.
- [x] I have disclosed any and all sources of reused code in the submitted repositories and have done my due diligence to meet its license requirements.
- [x] In case of acceptance, an invoice must be submitted and the payment will be transferred to the BTC/ETH/fiat account provided in the application.
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://grants.web3.foundation/docs/Support%20Docs/milestone-deliverables-guidelines).

Link to the application pull request: https://github.com/w3f/Grants-Program/pull/2004

### CODE REVIEW

**Comment by keeganquigley** [comment]
Thanks for the delivery @leonardocustodio someone will look at it in the next few days.

File: deliveries/polkadart-milestone_3.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+Documentation & guides & examples are spread through each package readme, you can find the link for each one here:
+
+https://github.com/leonardocustodio/polkadart/tree/main/packages/polkadart
+https://github.com/leonardocustodio/polkadart/tree/main/packages/polkadart_cli
+https://github.com/leonardocustodio/polkadart/tree/main/packages/polkadart_keyring
+https://github.com/leonardocustodio/polkadart/tree/main/packages/polkadart_scale_codec
+https://github.com/leonardocustodio/polkadart/tree/main/packages/ss58
+https://github.com/leonardocustodio/polkadart/tree/main/packages/substrate_bip39
+https://github.com/leonardocustodio/polkadart/tree/main/packages/substrate_metadata
```
Comment (semuelle) [inline deliveries/polkadart-milestone_3.md:143]:
```suggestion
- https://github.com/leonardocustodio/polkadart/tree/main/packages/polkadart
- https://github.com/leonardocustodio/polkadart/tree/main/packages/polkadart_cli
- https://github.com/leonardocustodio/polkadart/tree/main/packages/polkadart_keyring
- https://github.com/leonardocustodio/polkadart/tree/main/packages/polkadart_scale_codec
- https://github.com/leonardocustodio/polkadart/tree/main/packages/ss58
- https://github.com/leonardocustodio/polkadart/tree/main/packages/substrate_bip39
- https://github.com/leonardocustodio/polkadart/tree/main/packages/substrate_metadata
```

**Review by semuelle — CHANGES_REQUESTED** [review]
Thanks for the delivery, @leonardocustodio. Everything looks pretty good, I'm just missing a link to deliverable 0e.

**Comment by leonardocustodio** [comment]
Hello @semuelle , 
I have put the `demo` in gitignore (https://github.com/leonardocustodio/polkadart/blob/9b7ec8ef2e48b5d629a49c0f4276e4904b30f69e/.gitignore#L28) while I was working (because things were constantly changing) and forgot to take it out. I will do that as soon as I'm back, should be up by the end of the day, I will let you know, thank you for finding out.

**Comment by leonardocustodio** [comment]
Here, I've added the `examples` folder to the repository: https://github.com/leonardocustodio/polkadart/tree/main/examples

**Review by semuelle — APPROVED** [review]
Thanks for the update, @leonardocustodio. Your milestone is hereby accepted. Congrats! You can see my evaluation notes [here](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/evaluations/polkadart_3_semuelle.md). 

Looking forward to the next milestone.

**Comment by leonardocustodio** [comment]
Thanks @semuelle :rocket:

**Comment by meldien** [comment]
Hi @leonardocustodio 
Could you please add your full address on the invoice and send it again by the forms?
Many thanks in advance
Best wishes,
Melanie

**Comment by leonardocustodio** [comment]
Hello @meldien , sorry missed that. I've submitted a new form with the updated invoice.
Thank you,
Leonardo

**Comment by leonardocustodio** [comment]
@semuelle just making sure, should we merge this one?

**Comment by semuelle** [comment]
> @semuelle just making sure, should we merge this one?

Thanks @leonardocustodio, totally missed that!

**Comment by leonardocustodio** [comment]
@RouvenP thank you, appreciate it :)

**Pull request merged**